KMaina/Fast-Food-App
# #160298041 Create a Page for an Admin to See All Orders

#### What does this PR do?
Allows an admin to see all orders made by users and their status
#### Description of Task to be completed?
The admin should be able to

- View all orders made and their current status

- View an individual order by clicking on an order number

- View a customer's'order history by clicking on their name

- Change the status on a particular order depending on which level it is

#### How should this be manually tested?

- Clone the repo onto your local machine.
 
- Open a terminal and run `cd Fast-Food-App`
 
- run `git checkout -b ft-admin-order-all-160298041`
 
- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-admin-order-all-160298041`

- In the folder where you cloned the repo, you should open the UI folder.
 
- Open the admin-order-all.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298041
